jobs/e2e-test-cycloud.groovy

```diff
@@ -17,119 +17,149 @@ new Setup(steps
+      "DEPLOYER_USERNAME",
+      """Who asked to run this job, used to ping on slack.
+Typically not set manually, but rather by other jobs that call this one.""",
+      ""
 
 ).addStringParam(
-   "REVISION_DESCRIPTION",
-   """Set by the buildmaster to give a more human-readable description
-of the CYPRESS_GIT_REVISION, especially if it is a commit rather than a branch.
-Defaults to CYPRESS_GIT_REVISION.""",
-   ""
+      "REVISION_DESCRIPTION",
+      """Set by the buildmaster to give a more human-readable description
+of the GIT_REVISION, especially if it is a commit rather than a branch.
+Defaults to GIT_REVISION.""",
+      ""
 
 ).addStringParam(
-   "BUILDMASTER_DEPLOY_ID",
-   """Set by the buildmaster, can be used by scripts to associate jobs
+      "BUILDMASTER_DEPLOY_ID",
+      """Set by the buildmaster, can be used by scripts to associate jobs
 that are part of the same deploy.  Write-only; not used by this script.""",
-   ""
+      ""
 
 ).addBooleanParam(
-   "SET_SPLIT_COOKIE",
-   """IGNORE: This is a dummy parameter that is only here to avoid breaking the
+      "SET_SPLIT_COOKIE",
+      """IGNORE: This is a dummy parameter that is only here to avoid breaking the
+   communication with buildmaster""",
+      false
+
+).addStringParam(
+      "EXPECTED_VERSION",
+      """IGNORE: This is a dummy parameter that is only here to avoid breaking the
    communication with buildmaster""",
-   false
+      ""
 
 ).addStringParam(
-   "JOB_PRIORITY",
-   """The priority of the job to be run (a lower priority means it is run
+      "EXPECTED_VERSION_SERVICES",
+      """IGNORE: This is a dummy parameter that is only here to avoid breaking the
+   communication with buildmaster""",
+      ""
+
+).addStringParam(
+      "JOB_PRIORITY",
+      """The priority of the job to be run (a lower priority means it is run
 sooner). The Priority Sorter plugin reads this parameter in to reorder jobs
 in the queue accordingly. Should be set to 3 if the job is depended on by
 the currently deploying branch, otherwise 6. Legal values are 1
 through 11. See https://jenkins.khanacademy.org/advanced-build-queue/
 for more information.""",
-   "6"
+      "6"
 
 ).addStringParam(
-   "SKIP_TESTS",
-   """IGNORE: This is a dummy parameter that is only here to avoid breaking the
+      "SKIP_TESTS",
+      """IGNORE: This is a dummy parameter that is only here to avoid breaking the
    communication with buildmaster""",
-   ""
+      ""
 
 ).apply();
 
-
 // We set these to real values first thing below; but we do it within
 // the notify() so if there's an error setting them we notify on slack.
 NUM_WORKER_MACHINES = null;
 
 // Override the build name by the info that is passed in (from buildmaster).
-REVISION_DESCRIPTION = params.REVISION_DESCRIPTION ?: params.CYPRESS_GIT_REVISION;
+REVISION_DESCRIPTION = params.REVISION_DESCRIPTION ?: params.GIT_REVISION;
 E2E_URL = params.URL[-1] == '/' ? params.URL.substring(0, params.URL.length() - 1): params.URL;
```

Comment:
We might want to use a single def variable instead of calling params.URL multiple times.